bridge/engine/AudioForwarderReceiveJob.cpp

```diff
@@ -66,50 +66,40 @@ void AudioForwarderReceiveJob::decodeOpus(const memory::Packet& opusPacket)
     const uint32_t payloadLength = _packet->getLength() - headerLength;
     auto payloadStart = rtpPacket->getPayload();
 
-    if (decoder.getSequenceNumber() != 0)
+    if (decoder.hasDecoded() && _extendedSequenceNumber != decoder.getExpectedSequenceNumber())
     {
-        const auto expectedSequenceNumber = decoder.getSequenceNumber() + 1;
-
-        if (static_cast<int32_t>(decoder.getSequenceNumber() - sequenceNumber) >= 0)
+        int32_t lossCount = static_cast<int32_t>(_extendedSequenceNumber - decoder.getExpectedSequenceNumber());
+        if (lossCount <= 0)
         {
             logger::debug("Old opus packet sequence %u expected %u, discarding",
                 "OpusDecodeJob",
-                sequenceNumber,
-                expectedSequenceNumber);
+                _extendedSequenceNumber,
+                decoder.getExpectedSequenceNumber());
             return;
         }
-
-        if (sequenceNumber > expectedSequenceNumber)
+        else
```

Comment:
In my opinion if we already have a early return on the 1st `if` we should not create an `else` block. This increase the indentation and make it harder to read

Reply:
of course